Marlin/src/module/temperature.h

```diff
@@ -1383,6 +1383,9 @@ class Temperature {
       static void set_heating_message(const uint8_t, const bool=false) {}
     #endif
 
+    // Setter for raw_temps_ready (used by HAL)
+    static void set_raw_temps_ready(bool ready) { raw_temps_ready = ready; }
```

Comment:
obsolete?

Reply:
no, its required to trigger the watchdog refresh from core1 (via temperature code)